stable/gitlab-ee/README.md

```diff
@@ -0,0 +1,67 @@
+# GitLab Community Edition
+
+[GitLab Community Edition](https://about.gitlab.com/) is an application to code, test, and deploy code together. It provides Git repository management with fine grained access controls, code reviews, issue tracking, activity feeds, wikis, and continuous integration. 
+
+## Introduction
+
+This chart stands up a GitLab Community Edition install. This includes:
+
```

Comment:
You'll need to replace the Community Edition references with Enterprise Edition.